src/components/alphabets-vocabs/AlphabetsGrid.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Col } from "shards-react";
+
+const AlphabetsGrid = ({ alphabets }) => {
+  return (
+    <Col xs={4} md={3} lg={2}>
+      <h2>
+        <Link to={`/alphabets/${alphabets}`} className="text-decoration-none">
```

Comment:
A more readable option would be to use search parameters, i.e. <Link to={`/alphabets?letter=${alphabets}`}

Reply:
I will look it up in the next sprint